packages/Avalonia/AvaloniaSingleProject.targets

```diff
@@ -15,7 +15,6 @@
     <_AvaloniaIOSTarget Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">true</_AvaloniaIOSTarget>
```

Comment:
We can keep Tizen in this file. Avalonia.Tizen still can be used with single target project.